torchvision/csrc/io/image/image.cpp

```diff
@@ -1,6 +1,7 @@
 #include "image.h"
 
 #include <Python.h>
+#include <torch/library.h>
```

Comment:
I assume this is for `RegisterOperators`. Shouldn't we explicitly import the following instead?
```suggestion
#include <ATen/core/op_registration/op_registration.h>
```

Reply:
Ah good point. Changed.